lib/core/test/unit/Cardano/Wallet/DB/SqliteSpec.hs

```diff
@@ -34,29 +36,29 @@ import Data.ByteString
 import Data.Time.Clock
     ( getCurrentTime )
 import Test.Hspec
-    ( Spec, describe, it, shouldReturn )
+    ( Spec, beforeAll, beforeWith, describe, it, shouldReturn )
 
 spec :: Spec
-spec = do
+spec = beforeAll (newDBLayer Nothing) $ beforeWith cleanDB $ do
```

Comment:
good approach, instantiate db just ONCE before first spec item, and then clean db before EVERY spec item